woocommerce-gateway-stripe.php

```diff
@@ -120,7 +120,9 @@ public function __wakeup() {}
 			 * *Singleton* via the `new` operator from outside of this class.
 			 */
 			public function __construct() {
+				add_action( 'admin_menu', [ $this, 'add_payments_menu' ] );
```

Comment:
I think having its own controller would be good if we want to have each controller to map to a routing point. I can see us removing the onboarding wizard in the future, so having its own controller may make it easier to be removed (simply delete the file). The controller will have a simple html output that has a container for the react app to bind onto. The controller would also have some functions to load the react scripts, but that's just another way similar to `admin_menu` and `admin_enqueue_scripts` here, so that's not too much of a problem. 

The React-powered way skipped some of those and automatically bind the react app for us. Unfortunately, we don't need the menu so we needed extra code here to remove the submenu manually. 

It would be nice to take advantage of the wc-admin's react-powered page but I think having its own controller is also good. I can switch this to `WC_Stripe_Onboarding_Controller` (name to be decided).